petab/parameter_mapping.py

```diff
@@ -0,0 +1,368 @@
+"""Functions related to mapping parameter from model to parameter estimation
+problem"""
+
+import pandas as pd
+import numpy as np
+import libsbml
+import numbers
+import re
+from . import lint
+from . import core
+from typing import List, Tuple, Dict, Union
+import logging
+
+logger = logging.getLogger(__name__)
+
+# Parameter mapping for condition
+ParMappingDict = Dict[str, Union[str, numbers.Number]]
+# Parameter mapping for combination of preequilibration and simulation
+# condition
+ParMappingDictTuple = Tuple[ParMappingDict, ParMappingDict]
+# Same for scale mapping
+ScaleMappingDict = Dict[str, str]
+ScaleMappingDictTuple = Tuple[ScaleMappingDict, ScaleMappingDict]
+
+
+def get_optimization_to_simulation_parameter_mapping(
+        condition_df: pd.DataFrame,
+        measurement_df: pd.DataFrame,
+        parameter_df: pd.DataFrame = None,
+        sbml_model: libsbml.Model = None,
+        simulation_conditions=None,
```

Comment:
```suggestion
        simulation_conditions = None,
```

Reply:
Without typehints there shouldn't be any space afaik (pep8)